docs/cli/kops_create_cluster.md

```diff
@@ -120,6 +121,67 @@ kops create cluster [flags]
       --vpc string                       Set to use a shared VPC
   -y, --yes                              Specify --yes to immediately create the cluster
       --zones strings                    Zones in which to run the cluster
+=======
```

Comment:
looks like merge conflicts need to be resolved here. `make gen-cli-docs` should take care of this